
<TranslatedText
	xmlns:cws="http://schemas.cordys.com/cws/1.0"
	type="com.cordys.translation.TranslatedText"
	RuntimeDocumentID="50EB711A-AD5A-A1EC-B801-AFA55067AB87">
	<DocumentID>50EB711A-AD5A-A1EC-B801-AFA550676B87</DocumentID>
	<Name>50EB711A-AD5A-A1EC-B801-AFA550676B87</Name>
	<Description>Catch Exception</Description>
	<CreatedBy>sysadmin</CreatedBy>
	<CreationDate>2022-05-30T10:54:57.097</CreationDate>
	<Notes/>
	<Text>Catch Exception</Text>
	<Language>
		<uri
			id="50EB711A-AD5A-A1EC-B49C-3644C8A6EB84"
		/>
	</Language>
	<TextIdentifier>
		<uri
			id="50EB711A-AD5A-A1EC-B801-AFA55066EB87"
		/>
	</TextIdentifier>
</TranslatedText>
